src/System.Reflection.Emit/pkg/System.Reflection.Emit.pkgproj

```diff
@@ -2,15 +2,15 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Reflection.Emit.csproj">
-      <SupportedFramework>netcoreapp2.1;uap10.0.16299</SupportedFramework>
+      <SupportedFramework>netcoreapp2.1;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\src\System.Reflection.Emit.csproj" />
     <HarvestIncludePaths Include="ref/netstandard1.1" >
       <SupportedFramework>net45;netcoreapp1.0;netcore45;wpa81;$(AllXamarinFrameworks)</SupportedFramework>
     </HarvestIncludePaths>
     <HarvestIncludePaths Include="lib/netcore50" />
     <HarvestIncludePaths Include="lib/netstandard1.3" />
-    <InboxOnTargetFramework Include="monoandroid10;xamarinmac20" />
+    <InboxOnTargetFramework Include="$(AllXamarinFrameworks)" />
```

Comment:
This is breaking validation.  It's only inbox on `monoandroid10;xamarinmac20`.  
Since this is now part of `netstandard2.1` it means all Xamarin frameworks need to carry it inbox, so we should update the index to represent that.  @akoeplinger @mhutch is it on your radar to include these (and other) new facades that were brought into .NETStandard 2.1?

Reply:
@ericstj can you help to unblock this meanwhile? I assume there's a way to disable the validation temporarily?

Reply:
It wasn't on my radar - cc @marek-safar

Reply:
I think we had a discussion in dotnet/standard around this.  Let me find the issue.

Reply:
https://github.com/dotnet/standard/issues/1063 has the list of new facades that were added in ns2.1.  All ns2.1 supporting frameworks will need to provide these in order to handle loading libraries that were built against the packages before they were absorbed into ns2.1.